great_expectations/checkpoint/actions.py

```diff
@@ -782,6 +782,7 @@ class SNSNotificationAction(ValidationAction):
 
     type: Literal["sns"] = "sns"
 
+    notify_on: Literal["all", "failure", "success"] = "all"
```

Comment:
Do this class and the next not actually make use of this field? I don't see it used, and I don't see it on them in 0.18.